### PR TITLE
Update README_CN.md 

### PR DESCRIPTION
**Pull request by shenmo7192**
Update the readme chinese version up to date with english ver

### CODE REVIEW

**Comment by shenmo7192** [comment]
@ksco

File: README_CN.md [hunk 1]
```diff
@@ -77,7 +77,7 @@ box64 封装了 GTK，包括 gtk2 和 gtk3。
 关于 Steam 的注意事项
 ----
 
-请注意，Steam 是 32/64 位的混合体，所以还需要 box86 才能运行，因为客户端应用程序是 32 位的。它还使用 64 位本地服务器，但与大多数使用 libcef/chromium 的东西一样，它目前无法在 box64 上正常工作。所以，现在 box64 上暂时不能运行 Steam。
+请注意，Steam 是 32/64 位混合的应用，所以你需要 box86 才能运行，因为客户端应用程序是 32 位的。它还使用 64 位本地服务器，它的 `steamwebhelper` 无法被关闭（即使是在最小模式）而且会吃掉大量的内存。对于内存小于 6 GB 的机型，你将会需要创建swapfile来运行Steam.
```
Comment (ksco) [inline README_CN.md:80]:
```suggestion
请注意，Steam 是 32/64 位混合的应用，所以你需要 box86 才能运行，因为客户端应用程序是 32 位的。它还使用 64 位本地服务器，它的 steamwebhelper 无法被关闭（即使是在最小模式）而且会吃掉大量的内存。对于内存小于 6 GB 的机型，你将会需要创建 swapfile 来运行 Steam。
```

File: README_CN.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ Box64 可以在非 x86_64 Linux 系统（比如 ARM64）上运行 x86_64 Linux 
 
 由于 Box64 使用一些“系统”库的原生版本，如 libc、libm、SDL 和 OpenGL 等，因此很容易与大多数应用程序集成和使用，并且在许多情况下性能会相当不错。可以在[这里](https://box86.org/index.php/2021/06/game-performances/)查看一些性能测试的样例。
 
-Box64 集成了适用于 ARM64 平台的 DynaRec（动态重编译器），速度可以比纯解释模式快 5 到 10 倍。可以在[这里](https://box86.org/2021/07/inner-workings-a-high%E2%80%91level-view-of-box86-and-a-low%E2%80%91level-view-of-the-dynarec/)找到有关 DynaRec 工作原理的一些信息。
+Box64 集成了适用于 `ARM64` 和 `RV64` 平台的 DynaRec（动态重编译器），速度可以比纯解释模式快 5 到 10 倍。可以在[这里](https://box86.org/2021/07/inner-workings-a-high%E2%80%91level-view-of-box86-and-a-low%E2%80%91level-view-of-the-dynarec/)找到有关 DynaRec 工作原理的一些信息。
```
Comment (ksco) [inline README_CN.md:17]:
```suggestion
Box64 集成了适用于 ARM64 和 RV64 平台的 DynaRec（动态重编译器），速度可以比纯解释模式快 5 到 10 倍。可以在[这里](https://box86.org/2021/07/inner-workings-a-high%E2%80%91level-view-of-box86-and-a-low%E2%80%91level-view-of-the-dynarec/)找到有关 DynaRec 工作原理的一些信息。
```

**Review by ksco — APPROVED** [review]
LGTM, some minor changes are suggested.

**Comment by ptitSeb** [comment]
Thank you :)

**Comment by ksco** [comment]
Huh, I wonder why we don't squash PR's commits anymore?

**Comment by ptitSeb** [comment]
Oh, forgot to select it as squashed. my fault.

**Pull request merged**